src/github3/__about__.py

```diff
@@ -4,8 +4,8 @@
 __author__ = "Ian Stapleton Cordasco"
 __author_email__ = "graffatcolmingov@gmail.com"
 __license__ = "Modified BSD"
-__copyright__ = "Copyright 2012-2022 Ian Stapleton Cordasco"
-__version__ = "3.2.0"
+__copyright__ = "Copyright 2012-2023 Ian Stapleton Cordasco"
```

Comment:
We don't need to bump the copyright year unless there is significant changes. Likely we should just make this Copyright 2012 and be done with it